plugins/modules/change_request.py

```diff
@@ -227,13 +227,29 @@ def ensure_absent(module, table_client):
     return False, None, dict(before=None, after=None)
 
 
+def validate_params(params, change_request=None):
+    missing = []
+    if params["state"] == "closed":
+        missing.extend(
+            validation.missing_from_params_and_remote(
+                ("close_code", "close_notes"), params, change_request
+            )
+        )
+
+    if missing:
+        raise errors.ServiceNowError(
+            "Missing prequired arameters {0}".format(", ".join(missing))
```

Comment:
```suggestion
            "Missing required parameters {0}".format(", ".join(missing))
```

Reply:
Heh, I guess I accidentally hit `a` instead of `i` when I wanted to change "Missing parameters" to "Missing required parameters". Problems of a vim user ;)